src/urlsafe-base64-helper.js

```diff
@@ -0,0 +1,32 @@
+// Largely ported from https://github.com/RGBboy/urlsafe-base64
+
+'use strict';
+
+function encode(buffer) {
+   return buffer.toString('base64')
+    .replace(/\+/g, '-') // Convert '+' to '-'
+    .replace(/\//g, '_') // Convert '/' to '_'
+    .replace(/=+$/, ''); // Remove ending '='
+}
+
+function decode(base64) {
+   // Add removed at end '='
+  base64 += Array(5 - (base64.length % 4)).join('=');
+
+  base64 = base64
+    .replace(/-/g, '+') // Convert '-' to '+'
+    .replace(/_/g, '/'); // Convert '_' to '/'
+
+  // change from urlsafe-base64 since new Buffer() is deprecated
+  return Buffer.from(base64, 'base64').toString();
```

Comment:
In your earlier PR, you were doing `return Buffer.from(base64, 'base64');`: https://github.com/web-push-libs/web-push/pull/812/files#diff-86818fdabcd5c8f177c6c670df721fd6c2659a9d736388ca36e24a6482ad9433R21.

Reply:
Aah.
I'm clumsy
I will fix it right away